### PR TITLE
test(Wallet) Manage seed phrase account(imported and generated) and k…

### PR DESCRIPTION
**Pull request by Valentina1133**
Closed PR in old repo (https://github.com/status-im/status-desktop/pull/12097) and create a new one here
https://ethstatus.testrail.net/index.php?/cases/view/703030
https://ethstatus.testrail.net/index.php?/cases/view/703036
https://ethstatus.testrail.net/index.php?/cases/view/703514
Screenshot 2023-09-07 at 11 35 19

### CODE REVIEW

File: gui/screens/settings.py [hunk 1]
```diff
@@ -173,3 +173,33 @@ def get_community_info(self, name: str) -> UserCommunityInfo:
...
+        assert KeycardSettingsView().is_visible
+
+    @allure.step('Check that all keycard options displayed')
+    def all_keycard_options_available(self):
+        assert self._setup_keycard_with_existing_account_button.is_visible, f'Setup keycard with existing account not visible'
+        assert self._create_new_keycard_account_button.is_visible, f'Create new keycard button not visible'
+        assert self._import_restore_via_seed_phrase_button.is_visible, f'Import and restore via seed phrase button not visible'
+        self._scroll.vertical_scroll_to(self._import_from_keycard_button)
+        assert driver.waitFor(lambda: self._import_from_keycard_button.is_visible,
+                              10000), f'Import keycard button not visible'
```
Comment (StateOf-Vlado) [inline gui/screens/settings.py:201]: can we replace 10000 with configs.timeouts.UI_LOAD_TIMEOUT_MSEC ?

File: gui/screens/settings.py [hunk 2]
```diff
@@ -173,3 +173,33 @@ def get_community_info(self, name: str) -> UserCommunityInfo:
     def open_community_overview_settings(self, name: str):
         driver.mouseClick(self._get_community_item(name))
         return CommunitySettingsScreen().wait_until_appears()
+
+
+class KeycardSettingsView(QObject):
+
+    def __init__(self):
+        super(KeycardSettingsView, self).__init__('mainWindow_KeycardView')
+        self._scroll = Scroll('settingsContentBaseScrollView_Flickable')
```
Comment (StateOf-Vlado) [inline gui/screens/settings.py:182]: Scroll not imported

**Review by StateOf-Vlado — APPROVED** [review]

File: gui/screens/settings.py (hunk 2 above)
Comment (Valentina1133) [reply gui/screens/settings.py:183]: Added

File: gui/screens/settings.py (hunk 1 above)
Comment (Valentina1133) [reply gui/screens/settings.py]: changed to configs.timeouts.UI_LOAD_TIMEOUT_MSEC

**Pull request merged**